codex-rs/login/src/server.rs

```diff
@@ -50,6 +50,22 @@ pub struct LoginServer {
     pub shutdown_flag: Arc<AtomicBool>,
 }
 
+use std::mem::ManuallyDrop;
```

Comment:
Please consolidate imports at the top of the file. Exceptions are unit tests and conditionally compiled imports.